src/bankingAccount/Main.java

```diff
@@ -1,35 +1,221 @@
 package bankingAccount;
 
 
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args)
     {
+        Scanner input = new Scanner(System.in);
         //new bank with system properties
         Bank bank = new Bank();
+        System.out.println("Hello Sir, due to launching our bank and you are our first client you should create account to store it in our database and it's account number = 1");
+
+        //index for account no that increment in each account creation
+        int index = 1;
+
+        System.out.println("Please enter your name:");
+        String accountHolderName = input.nextLine();
+
+        System.out.println("Please enter your initial balance:");
+        double accountBalance = 0.0;
+        boolean validInput = false;
+        //loop and checking if the entered balance is valid balance or not
+        while (!validInput) {
+            try {
+                accountBalance = input.nextDouble();
+                input.nextLine(); // consume the newline character
+                validInput = accountBalance >= 0;
+                if (!validInput) {
+                    System.out.println("Please enter a valid positive balance");
```

Comment:
I think this line could be placed at the start of this method, and then this inner if wouldn't be needed.